client/flutter/pubspec.lock

```diff
@@ -251,5 +251,5 @@ packages:
     source: hosted
     version: "3.5.0"
 sdks:
-  dart: ">=2.4.0 <3.0.0"
+  dart: ">=2.5.0 <3.0.0"
```

Comment:
what impact if any does this have on device support?

Reply:
It doesn't have any impact on device support, it reflects the development enviroment.

Reply:
I confirm that updating the dart version doesn't affect device suppoert.

Reply:
Still, these changes are side effects from `flutter pub get` and should not be in this PR.